### ISSUE TITLE
Question: how to generate a solution notebook?

### ISSUE THREAD

**Issue by adimajo** [jupyter/nbgrader#1311]
Hi all,

I would like, parallel to the feedback generation mechanism, which is done per student, to generate a solution notebook, i.e. where the solution appears systematically (and eventually all tests) but without the nbgrader specific contents (cell types and points attributed to each question, etc.).

N.B.: I've also asked that question on Google Groups but there seems to be less activity + less specificity (https://groups.google.com/forum/#!topic/jupyter/77NdmnQLoBM).

Thanks in advance,
Adrien

**Comment by sigurdurb** [comment]
I don't think this had been implemented but it is a good idea we should mark it as a feature request. 
I guess since these notebooks are just json text files, you could just write a script that takes out the text you don't want and go from there.

**Comment by rmjarvis** [comment]
+1 on this feature request.  Currently I manually copy the solution code into a copy of they student version.  But it would be very nice if nbgrader could automate this step for me.
